docs/usage.md

```diff
@@ -64,6 +64,13 @@ do with the target path where we will mount the output directory during the buil
 
 See also `cachi2 generate-env --help`.
 
+### Use config parameters from file
+Cachi2 uses default configuration for several variables. See the [config.py](cachi2/core/config.py) module. 
+These parameters can be changed by using a configuration file:
+```shell
+ cachi2 --config-file config.yaml fetch-deps --source ./fzf gomod
+```
```

Comment:
I would leave docs out of this PR and add them separately

IMO this is a bit late to document the config, the `--config-file` param would be relevant while fetching deps already

Maybe configuration should have its own section in the README (and just link that section from the usage doc).

Reply:
ok, I'll leave that to a new PR